LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/Utils/ReceiveWindow.cs

```diff
@@ -5,5 +5,5 @@ namespace LoRaTools.Utils
 {
     using LoRaWan;
 
-    public record struct RX2ReceiveWindow(Hertz Frequency, DataRateIndex DataRate);
+    public record struct ReceiveWindow(DataRateIndex DataRate, Hertz Frequency);
```

Comment:
[nit] The combination of `DataRateIndex` and `Frequency` is also used in `RadioMetadata` as properties. I think using the `ReceiveWindow` there wouldn't make sense from a semantics-perspective but just bringing it up in case you missed it and you think it makes sense.

Reply:
That's an interesting point, but as you said, while the information seems identical, the fact that we have _receive_ in `ReceiveWindow` makes it odd to use/see on the transmission (TX) side of things (thus semantically different?). Moreover, the [radio metadata comes from the LNS protocol](https://lora-developers.semtech.com/build/software/lora-basics/lora-basics-for-gateways/?url=tcproto.html#radio-metadata), but `ReceiveWindow` maps to specific mentions of _receive windows_ in the LoRaWAN specification. Perhaps this needs more thought, but I'd rather not over-generalise for now.